src/main/java/com/fontys/api/service/UserService.java

```diff
@@ -19,7 +19,7 @@ public UserService(UserRepository userRepository)
     }
 
     @Transactional
-    public User createUser() { return userRepository.save(new User()); }
+    public User createUser(String UUID) { return userRepository.save(new User(UUID)); }
```

Comment:
UUID has to be unique

Reply:
hmmm, although you used the 'Unique' notation for the UUID in User.java, it still is possible to create users with the same UUID.